package.json

```diff
@@ -2,11 +2,14 @@
   "name": "iron-adventure-game",
   "version": "1.0.0",
   "description": "Adventure game \"capstone project\" for The Iron Yard Feb 2017 cohort",
-  "main": "",
+  "main": "server/app.js",
   "dependencies": {
     "angular": "^1.6.4",
     "angular-ui-router": "^0.4.2",
```

Comment:
for some reason, you have two packages named "body" and "parser":  I have a single one:

  "dependencies": {
    "body-parser": "^1.17.1",